lib/model/init.js

```diff
@@ -1,4 +1,5 @@
 const requestPromise = require('request-promise');
+const preferenceManager = require('../utils/preference-manager');
```

Comment:
It is better to reuse the [gitlab.js](https://github.com/AutolabJS/AutolabJS/blob/dev/tests/deployment_tests/gitlab.js) written for AutolabJS project. Please comment on the advantages and disadvantages of reuse.

Reply:
Alright. Let's keep your version for now.